_includes/specials/bech32/22-priority.md

```diff
@@ -0,0 +1,35 @@
+We've frequently mentioned the fee savings available to people spending
+segwit inputs, but we've never before mentioned that you don't need to
+take advantage of the savings.  If you want, you can pay the same fee
+you would've paid without segwit in order to possibly have your
+transaction confirm quicker (all other things being equal).  For some
+users, such as traders attempting arbitrage across exchanges, saving
+money may not be as important as getting transactions confirmed quicker
+for the same amount of money.
+
+To examine this idea, let's first generate a chart using Bitcoin Core's
+fee estimates that illustrates the potential fee savings available to
+creators of typical single-sig, one-input, two-output transactions:
```

Comment:
note: singlesig would be odd here yet fwiw STYLE.md stipulates multisig (as opposed to multi-sig)

Reply:
Yeah, it seems inconsistent, but that's the way I've always done it (and git grep in this repository shows lots of "single-sig" but no "singlesig").  I added a note to #171